src/components/biggive-heading/biggive-heading.tsx

```diff
@@ -1,64 +1,42 @@
 import { Component, Prop, h } from '@stencil/core';
+import { brandColour } from '../../globals/brand-colour';
+import { headingTag } from '../../globals/heading-tag';
+import { spacingOption } from '../../globals/spacing-option';
 
 @Component({
   tag: 'biggive-heading',
   styleUrl: 'biggive-heading.scss',
   shadow: true,
 })
 export class BiggiveHeading {
-  /**
-   * Space above component
-   */
-  @Prop() spaceAbove: number = 0;
-  /**
-   * Space below component
-   */
-  @Prop() spaceBelow: number = 0;
+  @Prop() spaceAbove: spacingOption = 2;
```

Comment:
:+1: nice use of constrained type, fixing [primitive obsession](https://moretestable.com/part1/lab03-find-primitive-obsession/).